qiskit/pulse/commands/command.py

```diff
@@ -115,5 +115,6 @@ def __hash__(self):
         return hash((type(self), self.duration, self.name))
 
     def __repr__(self):
-        return '%s(name=%s, duration=%d)' % (self.__class__.__name__,
-                                             self.name, self.duration)
+        return '%s(duration=%d, name="%s")' % (self.__class__.__name__,
```

Comment:
can we add the unit of duration, ie `dt` ?

Reply:
Same as above.